Library/Homebrew/extend/pathname.rb

```diff
@@ -432,6 +432,47 @@ def install_metafiles(from = Pathname.pwd)
     end
   end
 
+  sig { params(other_path: T.any(String, Pathname)).returns(T::Boolean) }
+  def parent_of?(other_path)
```

Comment:
I'd rather we didn't add more methods on `extend/Pathname`. Ideally I think we should remove all pathname extensions (or at least those that are there by default and not added later like e.g. `GitRepository`)

Reply:
Fair enough. Should I leave `parent_of?` here and have `shortened_brew_path` live in e.g. `Utils`? Or move both to `Utils`?

Reply:
Ideally both move to utils, I think.